format-specs/schema.json

```diff
@@ -64,22 +64,21 @@
             "bbox": {
               "type": "array",
               "description": "Bounding Box of the geometries in the file, formatted according to RFC 7946, section 5.",
-              "items": [
-                {
-                  "type": "number",
-                  "description": "The westmost constant longitude line that bounds the rectangle (xmin)."
-                },
-                {
-                  "type": "number",
-                  "description": "The minimum constant latitude line that bounds the rectangle (ymin)."
-                },
+              "items": {
+                "type": "number"
+              },
+              "oneOf": [
                 {
-                  "type": "number",
-                  "description": "The eastmost constant longitude line that bounds the rectangle (xmax)."
+                  "description": "2D bbox consisting of (xmin, ymin, xmax, ymax)",
+                  "minItems": 4,
+                  "maxItems": 4,
+                  "additionalItems": false
```

Comment:
The additionalItems doesn't add anything, you can remove this.

Reply:
Ah, yes, the additionalItems was only needed when using the prefixItems (and except that it would still had to be changed to items: false for the latest version)